tests/integration/test_integration_all_rust.py

```diff
@@ -590,983 +583,1003 @@ def teardown_module():
     kill_process(EP_SERVER)
 
 
-class TestRustWebPush:
-    """Test class for Rust Web Push."""
+@pytest.fixture(name="ws_url", scope="session")
+def fixture_ws_url() -> str:
+    """Return defined url for websocket connection."""
+    return f"ws://localhost:{CONNECTION_PORT}/"
+
+
+@pytest.fixture(name="broadcast_ws_url", scope="session")
+def fixture_broadcast_ws_url() -> str:
+    """Return websocket url for megaphone broadcast testing."""
+    return f"ws://localhost:{MP_CONNECTION_PORT}/"
+
 
-    # Max log lines allowed to be emitted by each node type
-    max_endpoint_logs = 8
-    max_conn_logs = 3
-    vapid_payload = {
+@pytest.fixture(name="vapid_payload", scope="session")
+def fixture_vapid_payload() -> dict[str, int | str]:
```

Comment:
Maybe define the return value as a type?

```suggestion
VapidPayload = dict[str, int | str]

@pytest.fixture(name="vapid_payload", scope="session")
def fixture_vapid_payload() -> VapidPayload:
```